packages/host/app/components/operator-mode/code-submode/playground/spec-search.gts

```diff
@@ -42,7 +42,6 @@ export default class SpecSearch extends Component<Signature> {
       this,
       () => this.args.query,
       () => this.args.realms,
-      { isLive: true },
```

Comment:
i don't think we want this to be live (which is different than being responsive)

Reply:
I was wrong, this is necessary...

Reply:
If a user creates a new spec via file modification, wudn't we want the new entry to appear?

Reply:
@tintinthong if you feel this is an important feature could you please add a test to this PR for it, ad you are more familiar with the expected functionality?

Reply:
sounds like a plan. this conversation has also sparked in me the idea that we need to share live searches in the store when they are the same query. I made this ticket for that https://linear.app/cardstack/issue/CS-8385/manage-live-search-subscriptions-in-the-store-to-optimize-for-search